lib/seahorn/BvOpSem2RawMemMgr.cc

```diff
@@ -308,21 +318,23 @@ Expr RawMemManager::coerce(Expr sort, Expr val) {
 /// \param[in] offsetBits number of bits at end of pointers reserved for byte
 ///            address
 /// \return symbolic value of the byte at the specified address
-Expr RawMemManager::extractUnalignedByte(Expr mem, PtrTy address,
+Expr RawMemManager::extractUnalignedByte(MemValTy mem, const PtrTy &address,
                                          unsigned offsetBits) {
   // pointers are partitioned into word address (high bits) and offset (low
   // bits)
-  PtrTy wordAddress = bv::extract(ptrSzInBits() - 1, offsetBits, address);
-  PtrTy byteOffset = bv::extract(offsetBits - 1, 0, address);
+  Expr wordAddress =
```

Comment:
`extract` and `concat` are now part of ALU. Need to update this code to use them.

Reply:
done